apps/src/templates/sectionProgressV2/AssignmentCompletionStatesBox.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import LegendItem from './LegendItem';
+import './section-progress-refresh.scss';
```

Comment:
```suggestion
import styles from './section-progress-refresh.scss';
```
Then use `className={styles.nameOfClassDefined}`. This adds a bit more compile time checking and also guarantees you are using the right class if there are duplicates names in two different CSS files.

Reply:
Done.